src/harden.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: all
+  name: Hardening tasks
+  become: yes
+  become_method: ansible.builtin.sudo
+  tasks:
+    - name: Harden system
+      ansible.builtin.include_role:
+        name: harden
+    - name: Install and configure login banner
+      ansible.builtin.include_role:
+        name: banner
```

Comment:
This role is already being applied via `base.yml`.  There is no point in applying it twice.

Reply:
I am re-applying it here, because the ansible hardening role changes it. So have to do it again.
see: https://github.com/cisagov/ubuntu-server-packer/pull/15/commits/858621aa6f61968ed7c07c5e76b291b05e0b15ae

Reply:
I could take it out of `base.yml` but I wanted to be as non-destructive as possible to align with our other packer repos.

Reply:
If you have to apply the banner role after the harden role, then you should remove the banner role from `base.yml` so that it is only applied once.

Reply:
see: [1876a33](https://github.com/cisagov/ubuntu-server-packer/pull/15/commits/1876a33f4ac0df751a1eaba178ee44006dc15905)

Reply:
Please also add a comment in `harden.yml` explaining why you are installing the banner after hardening, rather than in it's usual place in `base.yml`.

Reply:
My original quibble is satisfied, but please address @dav3r's request just above.

@dav3r - Feel free to resolve this conversation once you feel your request is addressed.

Reply:
Comments added: [34cac88](https://github.com/cisagov/ubuntu-server-packer/pull/15/commits/34cac8896a38c5130327db3f4023c8cc25269736)

Reply:
Thanks @bra1ncramp - I'll resolve this convo, though note that I started [a new one](https://github.com/cisagov/ubuntu-server-packer/pull/15/files#r1330312700) with some minor wording adjustments to your comment.